tests/containers/container_engine.pm

```diff
@@ -40,6 +40,10 @@ sub run {
 
     my $dir = "/root/DockerTest";
 
+    if (get_var('CONTAINERS_CGROUP_VERSION')) {
+        switch_cgroup_version($self, get_var('CONTAINERS_CGROUP_VERSION'));
```

Comment:
```suggestion
        $self->switch_cgroup_version(get_var('CONTAINERS_CGROUP_VERSION'));
```
Shouldn't it be this way? :thinking:

Reply:
> Shouldn't it be this way? 🤔

It's a function in containers::common.  I couldn't find a way to extend opensusebasetest so it can access `wait_boot()`.

Reply:
Ahh, I see. Yep.